src/main/resources/org/jetbrains/research/tasktracker/config/inspection_default.yaml

```diff
@@ -1 +1,6 @@
-mode: DEFAULT
\ No newline at end of file
+mode: DISABLE_SELECTED
+inspectionNames:
+  - "PyChainedComparisonsInspection"
+  - "PyPep8NamingInspection"
+  - "PyMissingOrEmptyDocstringInspection"
+  - "PyPep8Inspection"
```

Comment:
I am not sure if it works correctly in this example, since PyPep8Inspection inspection is a dummy inspection and calls Pep8ExternalAnnotator under the hood, you can see this strange error message then

Reply:
<img width="539" alt="Screenshot 2023-09-01 at 13 28 35" src="https://github.com/JetBrains-Research/tasktracker-3/assets/8993915/9837d38f-190c-4344-b5a3-a28ffaaff77d">

Reply:
I think we need to add an additional class that will store information inspection -> external annotator per language. We can add it only for Python for now and extend it in the future. Also, when you disable an inspection, you need to check if this inspection has an annotator and disable this annotator if it is necessary. What do you think?

Reply:
We can do it in a next pr if you wish